Organizing add section in docs (#32141)
home-assistant/home-assistant.github.io: source/_docs/organizing/floors.markdown

---
title: "Floors"
description: "Group your areas per floor"
---

> A floor in Home Assistant is a logical grouping of areas that are meant to match the physical floors in your home. Devices and entities
> cannot be assigned to floors directly but to areas. Floors can be used in automations and scripts as a target for actions. For example, to turn off all the lights on the downstairs floor when you go to bed.
> 

Reviewer: ```suggestion
A floor in Home Assistant is a logical grouping of areas meant to match your home's physical floors. Devices and entities
cannot be assigned to floors directly but to areas. Floors can be used in automations and scripts as a target for actions. For example, to turn off all the lights on the downstairs floor when you go to bed.

```

## Creating a floor

Follow these steps to create a new floor.

> 1. Go to {% my zones title="**Settings** > **Areas, labels & zones**" %} and select **Create floor**.

Reviewer: ```suggestion
1. Go to {% my areas title="**Settings** > **Areas, labels & zones**" %} and select **Create floor**.
```
